### PR TITLE
created department dropdown

### PR DESCRIPTION
**Pull request by manoj-surabathulla**
- created a department dropdown in the profile update screen
- created selected functionality for the dropdown
- selected value stored in the backend

### CODE REVIEW

File: src/pages/ProfileUpdate/index.tsx [hunk 1]
```diff
@@ -11,17 +11,21 @@ const ProfileUpdate = () => {
   const [values, setValues] = useState({
     employeeId: "",
     reportingTo: "",
-    department: "",
+    department: "Front End",
```
Comment (Vivekflyerssoft) [inline src/pages/ProfileUpdate/index.tsx:14]: ?

File: src/components/Dropdown/index.tsx [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
+const Dropdown = ({
+  label,
+  className,
+  fontType,
+  options,
+  onChange,
+  value,
+  name,
+  labelClassName,
+  textClassName,
```
Comment (Vivekflyerssoft) [inline src/components/Dropdown/index.tsx:12]: @ThreeShade @afrze Don't you think this is too much of customization?

File: src/pages/ProfileUpdate/index.tsx [hunk 3]
```diff
@@ -11,17 +11,21 @@ const ProfileUpdate = () => {
...
   });
 
   const department = [
+    // {
+    //   label: "select department",
+    //   value: "Select Department",
+    // },
```
Comment (Vivekflyerssoft) [inline src/pages/ProfileUpdate/index.tsx:21]: I hate commented code

File: src/components/Dropdown/index.tsx (hunk 2 above)
Comment (afrze) [reply src/components/Dropdown/index.tsx:12]: Yes, We normally give only few customization options. Have to remove `labelClassName`, `textClassName`

File: src/pages/ProfileUpdate/index.tsx (hunk 1 above)
Comment (afrze) [reply src/pages/ProfileUpdate/index.tsx]: Removed that, added string `Select Department`

**Review by afrze — APPROVED** [review]
LGTM

**Pull request merged**